src/components/redux/books/booksSlice.js

```diff
@@ -1,61 +0,0 @@
-/* eslint-disable no-param-reassign */
```

Comment:
- [ ] I don't know why you deleted this file, but `booksSlice` is here and it is a requirement for your fetch method in `redux/books.js` to update the state of `booksSlice`

Reply:
I deleted booksSlice and replaced it with books.js component.